scripts/brk2tmuxp/brk2tmuxp.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import sys
+
+import yaml
+
+
+def main():
+    """
+    reads a CML breakout labs.yaml file and returns a list of dict ready to be
+    translated into multiple JSON or YAML tmuxp session files.
+
+    By default, each telnet session will have its own window but if the `-p` or
+    `--panes` flag is set, then each telnet session will be in a pane in one
+    (and only one) window.
+
+    VNC sessions are ignored.
+    """
+    parser = argparse.ArgumentParser(
+        description="reads a CML breakout labs.yaml and generates a tmuxp session files"
+    )
+    parser.add_argument(
+        "-d",
+        "--brk-dir",
+        type=str,
+        default=".",
+        help="path to dir containing the breakout labs YAML files (default:: current directory)",
+    )
+    parser.add_argument(
+        "-f",
+        "--yaml-file",
+        type=str,
+        default="labs.yaml",
+        help="name of the 'labs' YAML file, (default to labs.yaml)",
+    )
+    parser.add_argument(
+        "-p",
+        "--panes",
+        help="if set, all telnet sessions will be in one window (default: each telnet session has its own window)",
+        action="store_true",
+    )
+    parser.add_argument(
+        "-l",
+        "--listen_addr",
+        default="::1",
+        help="specify the listen address (default: ::1)",
+    )
+    parser.add_argument(
+        "-s",
+        "--sleep",
+        type=float,
+        default="3",
+        help="sleep time (in seconds) before initiating telnet session (default: 3)",
+    )
+    parser.add_argument(
```

Comment:
Why would you do this?  Seems like since it's going to be read by another tool the format doesn't really matter.

Reply:
tmuxp accepts both formats JSON or YAML, tmuxp can also convert from YAML to JSON.
I added the option, thinking that some tmuxp users may like to have their configuration in JSON instead of YAML

Reply:
That's fine.  I was just curious.